src/phoenix/trace/llama_index/callback.py

```diff
@@ -212,7 +243,7 @@ def on_event_start(
             event_data["attributes"].update(
                 payload_to_semantic_attributes(event_type, payload),
             )
-
+        self._previous_non_exception_event_id = event_id
```

Comment:
since this is not a stack, it can only be cumulative. what if the following happens?
>span A begins
>>span B begins
>>>span C begins
>>>span C ends
>>
>>span B ends
>
>span A does something
>span A has an exception

then wouldn't the exception be attributed to span C instead of A?

Reply:
Good catch, mistake on my part.

Reply:
addressed in [7489389](https://github.com/Arize-ai/phoenix/pull/1449/commits/74893897ac3a8c149cde28d90900ec09ea0e9997)